publisher/publisher.go

```diff
@@ -369,3 +391,37 @@ func CreateTestingSignedSCT(req []string, k *ecdsa.PrivateKey, precert bool, tim
 	jsonSCT, _ := json.Marshal(jsonSCTObj)
 	return jsonSCT
 }
+
+func (pub *Impl) ProbeLogs() {
```

Comment:
Can you add a comment to this function?

Reply:
I think there should also be a small unit test for this function

Reply:
Done.